recipes/metatensor/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "0.1.5" %}
+{% set url_base = "https://github.com/lab-cosmo/metatensor/releases/download" %}
+
+{% set CMAKE_INSTALL_PREFIX = "-DCMAKE_INSTALL_PREFIX=$PREFIX" %}  # [not win]
+{% set CMAKE_INSTALL_PREFIX = "-DCMAKE_INSTALL_PREFIX=%PREFIX%/Library" %}  # [win]
+
+{% set RUST_TARGET = "-DRUST_BUILD_TARGET=x86_64-apple-darwin" %}  # [osx and x86_64]
+{% set RUST_TARGET = "-DRUST_BUILD_TARGET=aarch64-apple-darwin" %}  # [osx and arm64]
+
+{% set RUST_TARGET = "-DRUST_BUILD_TARGET=x86_64-pc-windows-msvc" %}  # [win and x86_64]
+{% set RUST_TARGET = "-DRUST_BUILD_TARGET=aarch64-pc-windows-msvc" %}  # [win and arm64]
+
+{% set RUST_TARGET = "-DRUST_BUILD_TARGET=x86_64-unknown-linux-gnu" %}  # [linux and x86_64]
+{% set RUST_TARGET = "-DRUST_BUILD_TARGET=aarch64-unknown-linux-gnu" %}  # [linux and aarch64]
+{% set RUST_TARGET = "-DRUST_BUILD_TARGET=powerpc64le-unknown-linux-gnu" %}  # [linux and ppc64le]
+
+{% set METATENSOR_CONFIG = "-DMETATENSOR_INSTALL_BOTH_STATIC_SHARED=OFF" %}
+{% set CMAKE_CONFIG = "-DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON" %}
+
+package:
+  name: metatensor
+  version: {{ version }}
+
+source:
+  url: {{ url_base }}/metatensor-core-v{{ version }}/metatensor-core-cxx-{{ version }}.tar.gz
+  sha256: 8ecac73604d79d7f3e46880cd14370d85b26ade3c9aa8ab9594678e8f3083a4f
+  patches:
+    - mktemp-mkdir.patch
+
+build:
+  number: 0
+  script: |
+    cmake -G Ninja {{ CMAKE_CONFIG }} {{ METATENSOR_CONFIG }} {{ RUST_TARGET }} {{ CMAKE_INSTALL_PREFIX }} .
+    cmake --build . --config Release --target install
```

Comment:
Please put this into a `build.sh` script. Otherwise, this way too much Jinja2 variables in here and makes it quite unreadable.

Reply:
Do I need to add both build.sh and build.bat then? My justification for putting it here is that otherwise I need to duplicate the exact same code for unix & windows